backend/app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\View\View;
+
+class LoginController extends Controller
+{
+    /**
+     * Get page login
+     */
+    public function login(): View {
+        return view('login');
+    }
+
+    /**
+     * Handle admin login
+     */
+    public function handleLogin (Request $request): RedirectResponse {
+       $credentials = $request->validate([
+            'username' => ['required'],
+            'password' => ['required'],
+       ]);
+
+       if (Auth::guard('admin')->attempt($credentials)){
+           $request->session()->regenerate();
+           return redirect()->intended('/students');
```

Comment:
@nhnhansgt 
Please change endpoint to the temporary route.
![image](https://github.com/saigon-tech/laravel-exercise/assets/51575975/cd75d9be-49f8-429b-848b-2dd7864606f1)

Example: 
```suggestion
           return redirect()->intended('/');
```